src/go/tests/BUILD.bazel

```diff
@@ -28,10 +28,7 @@ sh_test(
     data = [
         "@kubernetes_helm//:helm",
     ],
-    tags = [
-        "external",
-        "manual",
-    ],
+    tags = ["external"],
```

Comment:
I think we still need manual so that `bazel test //...` works on CloudTop without the extra setup of the sim VM (test_env stuff).

Reply:
This still works because https://github.com/googlecloudrobotics/core/blob/main/.bazelrc#L42.

If we add "manual" that means we need to explicitly list the tests in the test script, which I'm now avoid. But let me know if you prefer to be explicit.